styles/widgets/material/size-schemes/compact.less

```diff
@@ -0,0 +1,52 @@
+@MATERIAL_SIZE_POSTFIX: ".compact";
+
+// Font sizes
+@MATERIAL_BASE_FONT_SIZE: 13px;
+@MATERIAL_BASE_LINE_HEIGHT: 1.2857; // TODO doesn't know, what is it
```

Comment:
what about this TODO?

Reply:
It is magic. I do not want to change this, because I do not know what is this :)